packages/upgrade-tests/src/tests/automations.upgrade.test.ts

```diff
@@ -1,21 +1,30 @@
+import { AutomationStep } from "@budibase/types"
 import { BudibaseClient } from "../index"
-import { it } from "../utils/upgradeTest"
+import { shouldNotChange } from "../utils/upgradeTest"
 
-describe("Automation Upgrade Tests", () => {
+describe("automations", () => {
   let client: BudibaseClient
 
   beforeAll(async () => {
     client = await BudibaseClient.authenticated()
   })
 
-  it("should preserve automation counts", {
-    pre: async () => {
-      const automations = await client.automation.fetch()
-      return { automationCount: automations.length }
-    },
-    post: async ({ automationCount }) => {
-      const currentAutomations = await client.automation.fetch()
-      expect(currentAutomations.length).toBe(automationCount)
-    },
+  shouldNotChange("IDs", async () => {
+    const automations = await client.automation.fetch()
```

Comment:
I don't get these tests. Are they checking that the ids and other properties are not modified? Is this a real case scenario?